paying_for_college/disclosures/scripts/load_programs.py

```diff
@@ -89,18 +89,33 @@ def clean_string_as_string(string):
     return clean_str if clean_str.lower() not in NO_DATA_ENTRIES_LOWER else ''
 
 
+def standardize_rate(rate):
+    if rate and float(rate) > 1:
+        return unicode(float(rate)/100)
+    else:
+        return rate
```

Comment:
No need in either case. all the 'clean' commands just return strings, and string is later converted to Decimal on insertion. I just ran upload_programs to check, and all the rates load to Decimals without issue.